app/src/main/java/com/cornellappdev/android/eatery/TimeUtil.java

```diff
@@ -0,0 +1,35 @@
+package com.cornellappdev.android.eatery;
+
+import org.threeten.bp.ZoneId;
+
+public class TimeUtil {
+
+  private static TimeUtil instance;
+
+  public static TimeUtil getInstance() {
+    if (instance == null) {
+      instance = new TimeUtil();
+    }
+
```

Comment:
```
if (instance == null) instance = new TimeUtil();
return instance
```

Reply:
Google's style guide enforces braces for most blocks.